README.md

```diff
@@ -7,3 +7,46 @@ FAKE targets for getting projects off the ground ASAP
 ```
 Install-Package FSharp.FakeTargets
 ```
+
+### Usage
+
+Your `build.fsx` file might look like the following if you are using `NuGet` as your package manager.
+
+```fsx
+#r @"packages/FAKE/tools/FakeLib.dll"
+#r @"packages/FSharp.FakeTargets/tools/FSharpFakeTargets.dll"
+
+open Fake
+
+let private _overrideConfig (parameters : datNET.Targets.ConfigParams) =
+  { parameters with
+      Project     = "Pigs Can Fly"
```

Comment:
I feel like we should not encourage spaces in a project name... "PigsCanFly", maybe?